tests/mail/tests.py

```diff
@@ -85,6 +85,15 @@ def test_multiple_recipients(self):
         self.assertEqual(message['From'], 'from@example.com')
         self.assertEqual(message['To'], 'to@example.com, other@example.com')
 
+    def test_no_to_recipients(self):
```

Comment:
Something like `test_header_omitted_for_no_to_recipients` may be more descriptive.